globals.js

```diff
@@ -1,5 +1,6 @@
-module.exports = [
-  'SENTRY_DSN',
-  'GOOGLE_CLIENT_ID',
-  'GOOGLE_CLIENT_SECRET',
-];
+module.exports = {
+  SENTRY_DSN: JSON.stringify(process.env.SENTRY_DSN),
+  // https://console.developers.google.com/apis/credentials
+  GOOGLE_CLIENT_ID: JSON.stringify(process.env.GOOGLE_CLIENT_ID),
+  GOOGLE_CLIENT_SECRET: JSON.stringify(process.env.GOOGLE_CLIENT_SECRET),
```

Comment:
@brafdlog Need to register these values in Google, put them as *Repository Secrets*, and write them to *Env Vars* in our *Github Actions*.

Of course, I created them myself, maybe I can give you access to my registered app in Google, or maybe you register it again yourself.

Reply:
They changed the documentation a little, and it more detailed now. Anyway, I think I choose the right one:

![image](https://user-images.githubusercontent.com/17686879/99151964-8bb1a080-26a7-11eb-873e-ea24b4b369d1.png)

![image](https://user-images.githubusercontent.com/17686879/99151972-92401800-26a7-11eb-8b06-d0eb7703b9a5.png)